common/src/test/java/org/jboss/gm/common/utils/PluginUtilsTest.java

```diff
@@ -0,0 +1,138 @@
+package org.jboss.gm.common.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+import org.aeonbits.owner.ConfigCache;
+import org.apache.commons.io.FileUtils;
+import org.commonjava.maven.ext.common.ManipulationException;
+import org.gradle.api.logging.LogLevel;
+import org.gradle.tooling.internal.consumer.ConnectorServices;
+import org.jboss.gm.common.Configuration;
+import org.jboss.gm.common.rules.LoggingRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.RestoreSystemProperties;
+import org.junit.contrib.java.lang.system.SystemOutRule;
+import org.junit.rules.TemporaryFolder;
+import org.junit.rules.TestRule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class PluginUtilsTest {
+    @Rule
+    public final LoggingRule loggingRule = new LoggingRule(LogLevel.DEBUG);
+
+    @Rule
+    public final SystemOutRule systemOutRule = new SystemOutRule().enableLog().muteForSuccessfulTests();
+
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+
+    @Rule
+    public final TestRule restoreSystemProperties = new RestoreSystemProperties();
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Before
+    public void reset() {
+        // Reset the daemon between tests : https://discuss.gradle.org/t/stopping-gradle-daemon-via-tooling-api/16004/2
+        // Under 4.10 the daemon appears to cache Config values which corrupt the tests.
+        ConnectorServices.reset();
```

Comment:
For example, here is a `reset` without the `reload` but I did not see the original cause to be able to answer one way or another.

Reply:
I only appeared to need the reset here

Reply:
I believe it has something to do with the CLI vs running from a test. Nevermind then.